project.clj

```diff
@@ -9,8 +9,7 @@
                  [prismatic/schema "1.1.3"]
                  [org.clojure/data.csv "0.1.3"]
                  [schema-contrib "0.1.3"]
-                 [instaparse "1.4.3"]]
-  :main ^:skip-aot kixi.lapita
-  :target-path "target/%s"
+                 [instaparse "1.4.3"]
```

Comment:
I get this exception when I remove instaparse: https://gist.github.com/Eleonore9/df42f4a37c37927d4b7015408320043f